verticapy/sql/functions/string.py

```diff
@@ -36,6 +36,32 @@ def length(expr: SQLExpression) -> StringSQL:
     -------
```

Comment:
**For everything below:**

Same as above:

change verticapy.stats by verticapy.sql.functions as vpf
bring the comment of the function usage out the code (create a new block)
change TableSample by vDataFrame (so you have to change your import as well)
Try to follow PEP
Use a new line after the last import